File: .js/8kyu-How-good-are-you-really.js
// There was a test in your class and you passed it. Congratulations!
// But you're an ambitious person. You want to know if you're better than the average student in your class.

// You receive an array with your peers' test scores. Now calculate the average and compare your score!

// Return True if you're better, else False!

// Note:
// Your points are not included in the array of your class's points. For calculating the average point you may add your point to the given array!

const chai = require('chai');
const assert = chai.assert;

describe("Example Tests", function() {
  it("betterThanAverage([2, 3], 5) should return True", function() {
    assert.strictEqual(betterThanAverage([2, 3], 5), true);
  });
  
  it("betterThanAverage([100, 40, 34, 57, 29, 72, 57, 88], 75) should return True", function() {
    assert.strictEqual(betterThanAverage([100, 40, 34, 57, 29, 72, 57, 88], 75), true);
  });
  
  it("betterThanAverage([12, 23, 34, 45, 56, 67, 78, 89, 90], 9) should return False", function() {
    assert.strictEqual(betterThanAverage([12, 23, 34, 45, 56, 67, 78, 89, 90], 9), false);
  });
  
  it("betterThanAverage([41, 75, 72, 56, 80, 82, 81, 33], 50) should return False", function() {
    assert.strictEqual(betterThanAverage([41, 75, 72, 56, 80, 82, 81, 33], 50), false);
  });
  
  it("betterThanAverage([29, 55, 74, 60, 11, 90, 67, 28], 21) should return False", function() {
    assert.strictEqual(betterThanAverage([29, 55, 74, 60, 11, 90, 67, 28], 21), false);
  });
});

// my code

function betterThanAverage(classPoints, yourPoints) {
    let combinedScore=0;
    let classAverage=0;
    for (let i=0;i<classPoints.length;i++){
      combinedScore = combinedScore + classPoints[i];
    }
    classAverage = (combinedScore/classPoints.length);
    if (classAverage < yourPoints){
      return true;
    }  
    else{
      return false;
    }
  }

File: CodeWars/.cs/8kyu-HowGoodAreYouReally.cs
// There was a test in your class and you passed it. Congratulations!
// But you're an ambitious person. You want to know if you're better than the average student in your class.

// You receive an array with your peers' test scores. Now calculate the average and compare your score!

// Return True if you're better, else False!

// Note:
// Your points are not included in the array of your class's points. For calculating the average point you may add your point to the given array!

using System;
using System.Linq;
using NUnit.Framework;

[TestFixture]
public class Tests
{
  [Test]
  [TestCase(new int[]{2,3}, 5, ExpectedResult=true)]
  [TestCase(new int[]{100, 40, 34, 57, 29, 72, 57, 88}, 75, ExpectedResult=true)]
  [TestCase(new int[]{12,23,34,45,56,67,78,89,90}, 69, ExpectedResult=true)]
  public static bool FixedTest(int[] arr, int num)
  {
    return Kata.BetterThanAverage(arr, num);
  }
}

// my code

public class Kata
{
  public static bool BetterThanAverage(int[] ClassPoints, int YourPoints) => ((ClassPoints.Sum()+YourPoints) / ClassPoints.Length <= YourPoints);
}